README.md

```diff
@@ -2,69 +2,71 @@
+                               ┌──▼────────────▼──┐
+                               │ Data aggregation │
+                               │        API       │
+                               └─────────┬────────┘
+                                         │
+                                         │
+                                         │
+                                   ┌─────▼─────┐
+                                   │ SaltStack │
+                                   └─────┬─────┘
+                                         │
+                                         │
+                                         │
+                                         │
+                                ┌────────▼────────┐
+                                │ Network devices │
+                                └─────────────────┘
 
 ### Network CMDB
 
-The Network CMDB contains data relative to the business and is completely agnostic to the network OS. The models are designed to describe the objects themselves rather than the configuration from devices perspective. The idea is also to avoid any data duplication which could lead to mismatched configuration.
+The Network CMDB contains data relative to the business and is completely agnostic to the network OS.
 
-For instance, we represent the BGP session itself with two joined table describing peers: DeviceBGPSession <==> BGPSession <==> DeviceBGPSession.
-* DeviceBGPSession contains the local-as but not the peer-as, avoiding data duplication. The peer-as being the local-as of the other neighbor.
-* BGPSession contains all information peers have in common, like status (in production, maintenance etc...) or MD5 password
+The models are designed to describe the objects themselves rather than the configuration from device perspective. The idea is also to avoid any data duplication which could lead to configuration mismatchs.
+
+For instance, we represent the BGP session itself with two joined table describing peers: `DeviceBGPSession` <==> `BGPSession` <==> `DeviceBGPSession`.
+
+* `DeviceBGPSession` contains the `local-as` but not the `peer-as`, avoiding data duplication. The `peer-as` being the `local-as` of the other neighbor.
+* `BGPSession` contains all information peers have in common, like status (`in production`, `maintenance` etc...) or `MD5 password`
 
 ### Data Aggregation API
 
-This API aggregates data from their source of truths (Network CMDB or possibly any other data sources).
+This API aggregates data from their source of truth: Network CMDB or possibly any other data sources.
 
 Then, it computes these data to provide OpenConfig JSON for each device as an output.
 
-We use Ygot to validate the output against the OpenConfig YANG models.
+[ygot](https://github.com/openconfig/ygot) is used to validate the output against the OpenConfig YANG models.
 
-### Saltstack
+### SaltStack OpenConfig
 
 Salt will take OpenConfig data and convert them as Network configuration. We are using templates to do that.
```

Comment:
Salt will take OpenConfig data and convert them as Network configuration. We are using templates to do that.

> Salt will take OpenConfig data and convert them to device configurations using templates.